src/par/README.md

```diff
@@ -15,16 +15,363 @@ TritonPart is an open-source constraints-driven partitioner.  It can be used to
   - Embedding-aware partitioning
   
 ## Dependency
+
 We use Google OR-Tools as our ILP solver.  Please install Google OR-Tools following the [instructions](https://developers.google.com/optimization/install).
```

Comment:
Let's mention the `DependencyInstaller.sh` here as the preferred way to install the dependency. You can keep the official instructions as reference.